atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/service/SimpleTransactionService.java

```diff
@@ -140,6 +141,19 @@ private static ResilientCommitTimestampAtomicTable getDelegate(
                 store, encodingStrategy, acceptStagingReadsAsCommitted, metricRegistry);
     }
 
```

Comment:
The wiring in this class is super dangerous as it is prone to errors. Will probably be considering a more serious refactor of this class at some point.